manifests/m/Mihomo-Party/Mihomo-Party/1.5.11/Mihomo-Party.Mihomo-Party.locale.en-US.yaml

```
@@ -0,0 +1,35 @@
+# Created with WinGet Releaser using komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Mihomo-Party.Mihomo-Party
+PackageVersion: 1.5.11
+PackageLocale: en-US
+Publisher: Mihomo-Party

```

```suggestion
Publisher: mihomo-party-org
```

This value should match exactly what is written to Control Panel / Apps&Features post installation of the package